nj2k/src/org/jetbrains/kotlin/nj2k/ConversionsRunner.kt

```diff
@@ -80,7 +79,9 @@ object ConversionsRunner {
         RemoveRedundantQualifiersForCallsConversion(context),
         FilterImportsConversion(context),
         MoveInitBlocksToTheEndConversion(context),
-        AddElementsInfoConversion(context)
+        AddElementsInfoConversion(context),
+        ContinueStatementConversion(context),
```

Comment:
Please, do not add new conversions after `AddElementsInfoConversion` It is better to have all new conversions in before `TypeMappingConversion`. I will mention it in code later :)